tests/FileStorageGcsTest.php

```diff
@@ -15,14 +16,19 @@ class FileStorageGcsTest extends ClientTestCase
         'awsSecret' => TEST_S3_ROTATE_SECRET,
     ];
 
+
     private array $credentials;
 
     private array $rotateCredentials;
 
     public function setUp(): void
     {
-        $this->credentials = json_decode((string) TEST_GCS_KEYFILE_JSON, true, 512, JSON_THROW_ON_ERROR);
-        $this->rotateCredentials = json_decode((string) TEST_GCS_KEYFILE_ROTATE_JSON, true, 512, JSON_THROW_ON_ERROR);
+        /** @var mixed[] $credentials */
+        $credentials = json_decode((string) 'Xxx', true, 512, JSON_THROW_ON_ERROR);
```

Comment:
xxx ? tady by mely zustat puvodni konstanty

Reply:
Ježišmarjá, jsem dement, děkuju. Omylem jsem commitnul test (jestli ten phpstan fail je kvuli tomu, že je to ENV).